Makefile

```diff
@@ -1,4 +1,4 @@
 test:
-	emacs -batch -L . -l ./tests/highlight.el -l ./tests/indentation.el -f ert-run-tests-batch-and-exit
+	cask exec ert-runner -l jade-mode.el
```

Comment:
On my local and in [TCI](https://travis-ci.org/brianc/jade-mode/builds/73241865) `make test` gives me ```make: `test' is up to date.```

Applying the fix from [this SO question](http://stackoverflow.com/questions/3931741/why-does-make-think-the-target-is-up-to-date) solved it for me locally, could you make that change in this PR please?